### PR TITLE
Added Support for creating a Zabbix Inventory. It is possible to tag the

### PR DESCRIPTION
**Pull request by argo1984**
##### SUMMARY

Added Support for creating a Zabbix Inventory. It is possible to tag the hosts and add Locations etc.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

Module: zabbix_host

##### ANSIBLE VERSION

```
> ansible --version
ansible 2.3.0.0
  config file =
  configured module search path = Default w/o overrides
  python version = 2.7.13 (default, Mar 24 2017, 09:43:28) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```

##### ADDITIONAL INFORMATION

Usage Example in task:

```
- name: "Create a new host or update an existing host's info"
  local_action:
    module: zabbix_host
    server_url: "{{ zabbix_url }}"
    login_user: "{{ zabbix_api_user }}"
    login_password: "{{ zabbix_api_pass }}"
    host_name: "{{ zabbix_hostname }}"
    host_groups: "{{ zabbbix_hostgroup }}"
    link_templates: "{{ zabbix_link_templates }}"
    inventory_mode: automatic
    inventory_zabbix:
      tag: "{{ your_tag }}"
      alias: "{{ your_alias }}"
      notes: "Special Informations: {{ your_informations | default('None') }}"
      location: "{{ your_location }}"
      site_rack: "{{ your_site_rack }}"
      os: "{{ your_os }}"
      hardware: "{{ your_hardware }}"
      vendor: "{{ your_vendor }}"
    status: "{{ zabbix_host_status }}"
    state: "{{ zabbix_create_host }}"
    proxy: "{{ zabbix_proxy }}"
    interfaces: "{{ zabbix_interfaces }}"
  tags:
    - api
```

### CODE REVIEW

File: lib/ansible/modules/monitoring/zabbix_host.py [hunk 1]
```diff
@@ -90,6 +90,12 @@
         required: false
         default: None
         version_added: '2.1'
+    inventory_zabbix:
+        description:
+            - Add Facts for a zabbix inventory (e.g.: Tag)
+        required: false
+        default: None
+        version_added: '2.3'
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/zabbix_host.py]: `'2.4'`

File: lib/ansible/modules/monitoring/zabbix_host.py [hunk 2]
```diff
@@ -90,6 +90,12 @@
         required: false
         default: None
         version_added: '2.1'
+    inventory_zabbix:
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/zabbix_host.py:63]: Could you please add an example for this new option. Thanks

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/monitoring/zabbix_host.py (hunk 1 above)
Comment (argo1984) [reply lib/ansible/modules/monitoring/zabbix_host.py]: Hi,

i added an example. Sorry i didn't see your message before.

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
@eikef Could you please cast your eye over this PR, thanks

**Comment by D3DeFi** [comment]
Description of this option can use a link to [host inventory arguments](https://www.zabbix.com/documentation/3.2/manual/api/reference/host/object#host_inventory) as well as note that this needs to be paired with `inventory_mode: manual`. Same goes for example where it is used with `inventory_mode: automatic`.

Other than that it just needs [rebase](http://docs.ansible.com/ansible/latest/dev_guide/developing_rebasing.html) due to merge conflicts and will be good to ship.

**Comment by eikef** [comment]
Of the same opinion as @D3DeFi 

After a rebase it looks good to ship.

**Comment by eikef** [comment]
Thank you :-)

**Comment by eikef** [comment]
shipit

**Comment by D3DeFi** [comment]
shipit

**Pull request merged**